code/__DEFINES/mobs.dm

```diff
@@ -13,7 +13,7 @@
 //Blood levels
 #define BLOOD_VOLUME_MAX_LETHAL 2150
 #define BLOOD_VOLUME_EXCESS 2100
-#define BLOOD_VOLUME_MAXIMUM 2000
+#define BLOOD_VOLUME_MAXIMUM 1000 // SKYRAT EDIT - Blood volume balancing (mainly for Hemophages as nobody else really goes much above regular blood volume) - ORIGINAL VALUE: 2000
```

Comment:
you might want to touch those two defines above as well.

Reply:
I didn't touch those, as I don't want to touch non-modular stuff more than I should, and I don't want to make changes that affect much more than hemophages, which touching those values would.